### ISSUE TITLE
Add Ruby file extension to Ruby from Other Languages

### ISSUE THREAD

**Issue by Oisins** [ruby/www.ruby-lang.org#1343]
As a ruby beginner reading the help pages I would recommend adding something like: "In comparison to Python, Ruby files have the extension .rb" to the Ruby from Other Languages page.

**Comment by DanRathbun** [comment]
To which, I scratch my head in bemusement wondering, .. "What possible file extension could Python be using for Ruby files ?"

;) J/K

Perhaps "**In Ruby, script files use the `.rb` extension.**" is better ?

We cannot use the words "source files" because ruby libraries can be written in C / C++ (as well as other langauges,) and compiled to `.so`,`.dll`,`.dylib`, etc.

**Comment by Oisins** [comment]
Sounds good.

**Comment by myanch200** [comment]
Do we still want this added I couldn't see it mentioned in any of the pages? It is pretty obvious, but regardless.

**Comment by DanRathbun** [comment]
@kalpana-ghodeswar I believe this is a mistake by the newb.
The [Ruby from Other Languages](https://www.ruby-lang.org/en/documentation/ruby-from-other-languages/) HTML page is **not** a Trimble SketchUp authored document.
It is a ruby project help page and is outside the SketchUp API authority and scope of the SketchUp Extensibility Team.

**This issue should be closed as "Cannot Do".**

**Comment by kalpana-ghodeswar** [comment]
you are right Dan, I will let Chris know to close this as I dont have permissions to do so.
